src/aihawk_job_manager.py

```diff
@@ -420,7 +420,7 @@ def get_base_search_url(self, parameters):
         url_parts = []
         if parameters['remote']:
             url_parts.append("f_CF=f_WRA")
-        experience_levels = [str(i + 1) for i, (level, v) in enumerate(parameters.get('experience_level', {}).items()) if
+        experience_levels = [str(i + 1) for i, (level, v) in enumerate(parameters.get('experienceLevel', {}).items()) if
```

Comment:
@Nicolas-Maurer thanks for identifying it. 

upon reviewing the issue is with consistency in casing. Can you fix consistency? like making everything snake_case or camelCase

in both config yaml and also this file

Reply:
All in snake case please